src/averageAges.js

```diff
@@ -20,24 +20,49 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  let malesAverageAge = people.filter((person) => person.sex === 'm');
+
+  malesAverageAge = century !== undefined
+    ? malesAverageAge.filter(
+      (person) => Math.ceil(person.died / 100) === century
+    )
+    : malesAverageAge;
```

Comment:
use `const` and do it in 1 condition.
use ternary, but don't check if century is undefined, go with `century ? a : b`